terminal/terminal.go

```diff
@@ -150,6 +167,34 @@ func (t *Term) Println(prefix, str string) {
 	fmt.Fprintf(t.stdout, "%s%s\n", prefix, str)
 }
 
+// SubstitutePath substitues directory to soruce file.
+//
+// Ensures that only directory is substitued, for example:
+// substitute from `/dir/subdir`, substitute to `/new`
+// for file path `/dir/subdir/file` will return file path `/new/file`.
+// for file path `/dir/subdir-2/file` substitution will not be applied.
+//
+// If more than one substitution rule is defiend, the rules are applied
+// in the order they are defined, first rule that matches is used for
+// substitution.
+func (t *Term) SubstitutePath(path string) string {
+	separator := string(os.PathSeparator)
```

Comment:
> Could you please provide a bit more information, on what exactly will not work on windows?

For example your line 191 uses strings.HasPrefix to compare beginning of two paths. But windows paths are case insensitive. So the user can provide either from="c:\dir\subdir" or from="C:\DIR\SUBDIR" (even from="c:\DiR\SuBdIr" should work) and it should work.